tasks/main.yml

```diff
@@ -1,6 +1,14 @@
 ---
 # Tasks file for geonode
 
+- fail:
```

Comment:
The problem is that previous playbooks might not have those variables set, so they will fail right away